appinventor/appengine/src/com/google/appinventor/server/project/youngandroid/YoungAndroidProjectService.java

```diff
@@ -114,6 +114,9 @@
   // host[:port] to use for connecting to the build server
   private static final Flag<String> buildServerHost =
       Flag.createFlag("build.server.host", "localhost:9990");
+  // host[:port] to tell build server app host url
+  private static final Flag<String> appengineHost =
+      Flag.createFlag("appengine.host", "localhost:8080");
```

Comment:
@wanddy Perhaps the default here should be "" instead of localhost:8080. In theory this default will never get used because the appengine-web.xml file sets the property to "". This is why I was confused about whether or not the code would work.

-Jeff

P.S. Sorry for the delay in all of this. Been busy :-)